src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -22,7 +22,7 @@ static ucs_config_field_t uct_cuda_copy_iface_config_table[] = {
      "Max number of event completions to pick during cuda events polling",
      ucs_offsetof(uct_cuda_copy_iface_config_t, max_poll), UCS_CONFIG_TYPE_UINT},
 
-    {"MAX_EVENTS", "1024",
+    {"MAX_EVENTS", "inf",
```

Comment:
yes.   It required more events because now it can have fragmentation at both sender side and receiver side.  they are already insufficient for tests like osu_bw, osu_bibw with larger window

Reply:
yes. related change. we need more event objects because of fragmentation at both the sender and receiver.